src/MongoDB.Driver.Core/Core/Clusters/MultiServerCluster.cs

```diff
@@ -160,13 +165,27 @@ public override void Initialize()
                 {
                     _openedEventHandler(new ClusterOpenedEvent(ClusterId, Settings, stopwatch.Elapsed));
                 }
+
+                MonitorServersAsync().ConfigureAwait(false);
```

Comment:
This used to happen earlier, but I think this is a better point in time to start the background heartbeat task.

Reply:
Ack.